CREDITS.md

```diff
@@ -1,42 +1,62 @@
-# Forerunners
+# Credits
```

Comment:
@erlend-sh I'd like to get your review on the updated credits. We can change this any time so it doesn't need to block the PR if you want to get to this later.

I added all the Git commiters that weren't previously in the list ( we'll probably want to automate this later ), and now the game will parse the Credits file to render the in-game credits so we don't have to maintain two lists:

![image](https://user-images.githubusercontent.com/25393315/223842002-d8cb63a9-e16d-421f-940b-b096c05993ef.png)

I wasn't sure if there's more criteria for ending up in the credits or not.

Reply:
Ah, just saw your comment. 👍

Reply:
That’s excellent 👌 I think the only ones missing are some artists, so I’ll cover that in my second pass.